numpy/core/src/umath/scalarmath.c.src

```diff
@@ -167,7 +174,7 @@ static NPY_INLINE int
     }
 #if @neg@
     else if (b == -1 && a == NPY_MIN_@NAME@) {
-        *out = a / b;
+        *out = NPY_MIN_@NAME@;
```

Comment:
Ok it seems like its not covered here as well which is weird, let me check. ref: https://github.com/numpy/numpy/pull/21648#discussion_r895173501

Reply:
I can confirm it's hitting in my machine.

Reply:
In which test? Maybe somehow it is skipped in CI?

Reply:
`numpy/core/tests/test_umath.py::TestDivisionOverflows::test_overflows[floordiv-int8-int8]`

https://github.com/numpy/numpy/pull/21507/files#diff-ddb5d5f0dc988a0ab77107bb335499fd1ae477f73343f244e3c4d039bdca5166R786 Line to be precise